tracer/src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -87,6 +93,10 @@ public AspNetCoreDiagnosticObserver(Tracer tracer, Security security, Iast.Iast
 
         private Iast.Iast CurrentIast => _iast ?? Iast.Iast.Instance;
 
+        private LiveDebugger CurrentLiveDebugger => _liveDebugger ?? LiveDebugger.Instance;
+
+        private SpanCodeOriginManager CurrentCodeOriginManager => this._spanOrigin ?? SpanCodeOriginManager.Instance;
```

Comment:
nit: don't use `this`
```suggestion
        private SpanCodeOriginManager CurrentCodeOriginManager => _spanOriginManager ?? SpanCodeOriginManager.Instance;
```

Reply:
Fixed in a5a2c085c57f23a246a2b781dec1bc9354502a16